server/server.js

```diff
@@ -14,7 +17,9 @@ app.use(express.urlencoded({ extended: true }));
 app.use('/', express.static('client/public'));
 
 app.get('/api/photos', (req, res) => {
-  res.send('Hello World');
+  db.getAllPhotos((err, result) => {
+    res.send(result);
+  });
```

Comment:
In what format do you whant to send the data in your request ? 
take a look on that -> 
res.send uses content-type:text/html
res.json uses content-type:application/json

Reply:
@ingdjason Good catch! Refactored to res.json()